p2-repository/org.eclipse.collections/pom.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
```

Comment:
Oh! This is the pom _inside_ the p2-repository, not the pom _for_ the p2-repository module.
That's why I was confused, because there were 2 poms under the same module. 
I am still learning about p2 repository, so I will trust your judgement in this regard :smiley: